docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -135,10 +135,11 @@ Each component in a single devfile must have a unique name and use one of the ob
 
```

Comment:
- Suggestion: The content from L7 to L124 can go as a separate concept topic that you can link to this procedure under additional resources (minimalism principles). See what suits best for the user. As a reader, it is not comfortable for me to go through a long table when I am looking for a procedure.